jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2511 @@
+{
```

Comment:
<p>We usually don't write titles like "example pipeline" or "tutorial" or something. We rather write "Deconvolving blabla data using MuSiC" or something like this. For a more pleasant reading experience we avoid these words. Can you find a better title?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>You guys can edit the titles later to have it more in your style.</p>